src/main/scala/de/hpi/callcenterdashboard/DatabaseConnection.scala

```diff
@@ -37,5 +39,33 @@ class DatabaseConnection {
       case e: Throwable => e.printStackTrace()
     }
   }
+  def printCustomerResults (resultSet : java.sql.ResultSet): Unit = {
+    while (resultSet.next()) {
+      println(resultSet.getString("NAME"))
+      println(resultSet.getString("STRASSE") + ", " + resultSet.getString("PLZ") + " " + resultSet.getString("ORT") + ", " + resultSet.getString("LAND"))
+      println(resultSet.getString("BRANCHE") + " " + resultSet.getString("KUNDENGRUPPE"))
+    }
+  }
+  def printCustomersBy(kdnr: String = "", name: String = "", plz: String = ""): Unit = {
+    try {
+      // create the statement, and run the select query
+      val statement = connection.get.createStatement()
+      if (kdnr != "") {
+        val resultSet = statement.executeQuery("SELECT * FROM SAPQ92.KNA1_HPI WHERE LOWER(KUNDE) LIKE LOWER('" + kdnr + "') LIMIT 100")
```

Comment:
Den User, der uns SQL injected würde ich feiern :D
Wir können das ja in der Eingabe abfangen. Oder du nutzt eine _real_escape_string_ Funktion. (Gibts sowas für HANA?)